include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -164,6 +164,20 @@ using __fpga_selector = sycl::INTEL::fpga_selector;
 #    endif
 #endif // _ONEDPL_FPGA_DEVICE
 
+using __target =
```

Comment:
May be better to use something like this?

#if __LIBSYCL_VERSION >= 50400
    using __target = sycl::target;
#else
    using __target = sycl::access::target;
#endif

__fpga_emulator_selector type defined in this file like this syntax.

Reply:
I rather leave the code as is, because there is only one row within the macro guards comparing to the section with `__fpga_emulator_selector`

Reply:
ok